docs/docker_container.md

```diff
@@ -215,7 +215,7 @@ Here the numerical values depict the version number of the model.
 | `"batch_size"` | `integer / "auto"` | Optional. By default, the batch size is derived from the model, defined through the OpenVINO Model Optimizer. `batch_size` is useful for sequential inference requests of the same batch size.<br><br>Some models, such as object detection, don't work correctly with the `batch_size` parameter. With these models, the output's first dimension doesn't represent the batch size. You can set the batch size for these models by using network reshaping and setting the `shape` parameter appropriately.<br><br>The default option of using the Model Optimizer to determine the batch size uses the size of the first dimension in the first input for the size. For example, if the input shape is `(1, 3, 225, 225)`, the batch size is set to `1`. If you set `batch_size` to a numerical value, the model batch size is changed when the service starts.<br><br>`batch_size` also accepts a value of `auto`. If you use `auto`, then the served model batch size is set according to the incoming data at run time. The model is reloaded each time the input data changes the batch size. You might see a delayed response upon the first request.<br>  ||
-| `"layout" `| `json / string` | `layout` is optional argument which allows to change the layout of model input and output tensors. To add the transposition step, specify tensor layout and native model layout separated by `:` (colon). Example: `NHWC:NCHW` means that user will send input data in `NHWC` layout but model is in `NCHW` layout.<br><br>When specified with single string value - layout change is only applied to single model input. To change multiple model inputs or outputs, you can specify json object with mapping, such as: `{"input1":"NHWC","input2":"NHWC","output1":"NHWC"}`.<br><br>If not specified, layout is inherited from model. ||
+| `"layout" `| `json / string` | `layout` is optional argument which allows to change the layout of model input and output tensors. To add the transposition step, specify tensor layout and native model layout separated by `:` (colon). Example: `NHWC:NCHW` means that user will send input data in `NHWC` layout but model is in `NCHW` layout.<br><br>When specified with single string value - layout change is only applied to single model input. To change multiple model inputs or outputs, you can specify json object with mapping, such as: `{"input1":"NHWC:NCHW","input2":"HWN:NHW","output1":"CN:NC"}`.<br><br>If not specified, layout is inherited from model. ||
```

Comment:
it is not clear what should be set in the first part of the layout and what after the colon. Maybe a format like "<target_layout:source layout>". Target layout refers to the expected format of the serving input data while the source layout defines the original model.
It should be also added info about the format without a colon - it sets the layout to be used by the model server to change the batch size. That might be needed if the model has batch size on a position other then the first dimension and the layout was not set in the model via the model optimizer tool.

Reply:
The `:` (colon) is well described in another document. I think there is no need to duplicate. Maybe link to the separate document would be sufficient?